src/ffi.rs

```diff
@@ -1,6 +1,10 @@
 //! Package ffi contains bindings to underlying C API and thin abstraction for managing C handles.
 #![allow(warnings)]
 #![allow(unaligned_references)]
+
+#[cfg(bindgen)]
+include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
```

Comment:
If you want we could have separate bindings files for Linux, Windows, and Mac. It's probably unnecessary but would help non-bindgen build work better on all platforms. It would mean that bumping the Speech SDK version would require a bindgen run on each of those platforms.

Reply:
I am thinking if we need _skip binding_ flag at all. This was implemented because I wanted to disable changes in binding when doing some unrelated changes which happened quite a lot. Including generated bindings into source code seemed like good a=idea at the beginning (I thought I will have better overview and visibility what is going deep down under the hood) but it turned out to be rather awkward from many reasons. So moving this into target folder is definitely good step (apart from custom definitions in `src/ffi.rs`). So maybe I would remove env var `MS_COG_SVC_SPEECH_SKIP_BINDGEN`, feature `bindgen `and make it simpler. Do you think this is good idea? Any thoughts why we should keep MS_COG_SVC_SPEECH_SKIP_BINDGEN?

Reply:
I think we could get away with removing it entirely, the only problem is that it does force users to have libclang available, but I think most -sys crates require that anyway as it's the recommended approach in the bindgen docs so it's probably fine. I do think we need to keep `src/ffi/bindings.rs` though as otherwise docs.rs won't be able to build the docs, and it will make rust-analyzer a bit worse.

Reply:
I did not realise we still need it for docs.rs (which is such a pain in general... :) ), let's keep `MS_COG_SVC_SPEECH_SKIP_BINDGEN` then and let's make 3 separate binding files per each supported platform. thanks!

Reply:
docs.rs sets a different environment variable that we use to skip all the build steps anyway, so we could still remove the `MS_COG_SVC_SPEECH_SKIP_BINDGEN` env var, but I don't mind. I've also realised I just lied about rust-analyzer, so I think while we need to keep bindings.rs, it's probably pointless having different ones for windows, mac, and linux, I can't see when they'd ever get used (unless you want to keep SKIP_BINDGEN for some reason).